src/tape/TapePlayer.cpp

```diff
@@ -30,20 +31,21 @@ TapePlayer::TapePlayer(const char* path)
 	playbackState = std::make_unique<PlaybackState>(pathBin.c_str());
 
 	yamlRoot = YAML::LoadFile(pathYaml);
-	yamlRecording = yamlRoot["recording"];
 
-	if (yamlRoot[RGL_VERSION]["major"].as<int>() != RGL_VERSION_MAJOR ||
-	    yamlRoot[RGL_VERSION]["minor"].as<int>() != RGL_VERSION_MINOR) {
-		throw RecordError("recording version does not match rgl version");
+	auto version = getTapeCall(findFirst({RGL_VERSION}).value()).getArgsNode();
+	int versionLinear = 1'000'000 * version[0].as<int>() + 1'000 * version[1].as<int>() + version[2].as<int>();
+	int lastTapeUpdate = 16 * 1'000 + 3;
+	if (versionLinear < lastTapeUpdate) {
+		throw RecordError("Unsupported (outdated) Tape format");
```

Comment:
It would be nice to inform which version of tape format is read and which version is required.